public/ticketmanager.php

```diff
@@ -398,37 +440,75 @@
                 }
                 if ($modeHardcore) {
-                    echo "<b><a href='/ticketmanager.php?g=$gameIDGiven&u=$assignedToUser&f=$gamesTableFlag&t=" . ($ticketFilters & ~(1 << 12)) . "'>*Hardcore</a></b> | ";
+                    echo "<b><a href='$standardFilterUrl" . ($ticketFilters & ~(1 << 12)) . "'>*Hardcore</a></b> | ";
                 } else {
-                    echo "<a href='/ticketmanager.php?g=$gameIDGiven&u=$assignedToUser&f=$gamesTableFlag&t=" . ($ticketFilters | (1 << 12)) . "'>Hardcore</a> | ";
+                    echo "<a href='$standardFilterUrl" . ($ticketFilters | (1 << 12)) . "'>Hardcore</a> | ";
                 }
                 if ($modeSoftcore) {
-                    echo "<b><a href='/ticketmanager.php?g=$gameIDGiven&u=$assignedToUser&f=$gamesTableFlag&t=" . ($ticketFilters & ~(1 << 13)) . "'>*Softcore</a></b>";
+                    echo "<b><a href='$standardFilterUrl" . ($ticketFilters & ~(1 << 13)) . "'>*Softcore</a></b>";
+                } else {
+                    echo "<a href='$standardFilterUrl" . ($ticketFilters | (1 << 13)) . "'>Softcore</a>";
+                }
+                echo "</div>";
+
+                // Active Dev Filters
+                echo "<div>";
+                echo "<b>Dev Status:</b> ";
+
+                if ($devInactive) {
+                    echo "<b><a href='$standardFilterUrl" . ($ticketFilters & ~(1 << 14)) . "'>*Inactive</a></b> | ";
+                } else {
+                    echo "<a href='$standardFilterUrl" . ($ticketFilters | (1 << 14)) . "'>Inactive</a> | ";
+                }
+                if ($devActive) {
+                    echo "<b><a href='$standardFilterUrl" . ($ticketFilters & ~(1 << 15)) . "'>*Active</a></b> | ";
+                } else {
+                    echo "<a href='$standardFilterUrl" . ($ticketFilters | (1 << 15)) . "'>Active</a> | ";
+                }
+                if ($devJunior) {
+                    echo "<b><a href='$standardFilterUrl" . ($ticketFilters & ~(1 << 16)) . "'>*Junior</a></b>";
                 } else {
-                    echo "<a href='/ticketmanager.php?g=$gameIDGiven&u=$assignedToUser&f=$gamesTableFlag&t=" . ($ticketFilters | (1 << 13)) . "'>Softcore</a>";
+                    echo "<a href='$standardFilterUrl" . ($ticketFilters | (1 << 16)) . "'>Junior</a>";
                 }
                 echo "</div>";
 
+                // Resolved By Filter
+                if ($closedTickets || $resolvedTickets) {
+                    echo "<div>";
+                    echo "<b>Resolved By:</b> ";
+
+                    if ($notAuthorStatus) {
+                        echo "<b><a href='$standardFilterUrl" . ($ticketFilters & ~(1 << 17)) . "'>*Not Author</a></b> ";
+                    } else {
+                        echo "<a href='$standardFilterUrl" . ($ticketFilters | (1 << 17)) . "'>Not Author</a> ";
```

Comment:
I definitely like this better than the previous text, but I'm worried that "Author" is ambiguous between the achievement author and the ticket author. Maybe "Not Developer" to match the column in the list, or "Not Creator" to match the "Created by <user>" shown on the achievement page?